zebrad/src/components/mempool.rs

```diff
@@ -82,6 +82,9 @@ pub struct Mempool {
     /// Allows checking if we are near the tip to enable/disable the mempool.
     #[allow(dead_code)]
     sync_status: SyncStatus,
+
+    /// Access the state of the current blockchain.
+    state: State,
```

Comment:
State is needed to get the blockchain tip height.

Reply:
This is a LOT better and it is very easy/convenient to use in this case, thank you very much. I made the changes to use `LatestChainTip` at https://github.com/ZcashFoundation/zebra/pull/2774/commits/fc47b1403f75e8030e07d3e94c696af50f436f9c